src/AccountServices.js

```diff
@@ -0,0 +1,147 @@
+var RemoteRoom = require("matrix-appservice-bridge").RemoteRoom;
+var RemoteUser = require("matrix-appservice-bridge").RemoteUser;
+var Twitter    = require('twitter');
+var log        = require('npmlog');
+var OAuth      = require('oauth');
+var TwitterHandler = require('./TwitterHandler.js').TwitterHandler;
+
+
+
+/**
+ * Construct a AccountServices.
+ * This class is a handler for conversation between users and the bridge bot to
+ * link accounts together
+ *
+ * @class
+ * @extends {external:TwitterHandler}
+ *
+ * @param  {matrix-appservice-bridge.Bridge}   bridge
+ * @param  app_auth OAuth authentication information
+ * @param  app_auth.consumer_key Twitter consumer key
+ * @param  app_auth.consumer_secret Twitter consumer secret
+ * @param  {TwitterDB}       storage
+ * @param  {MatrixTwitter}   twitter
+ */
+var AccountServices = function (bridge, app_auth, storage, twitter) {
```

Comment:
/me refers you to https://github.com/matrix-org/matrix-appservice-gitter/pull/8#discussion_r69425769